src/app/home/home.component.ts

```diff
@@ -18,6 +18,7 @@ import {
 
 export class HomeComponent implements OnInit {
   weather: IWeather;
+  weathers: IWeather;
```

Comment:
What do you think if we use `extendedForecast` or something like that instead of `weathers` ?

Reply:
It was changed. Thanks